kv/union_store.go

```diff
@@ -80,7 +79,6 @@ type unionStore struct {
 func NewUnionStore(snapshot Snapshot) UnionStore {
```

Comment:
Can you remove the argument of the function?

Reply:
It is used in `NewBufferStore`.